tfhe/src/core_crypto/commons/math/random/mod.rs

```diff
@@ -113,16 +113,28 @@ impl<T: UnsignedInteger> DynamicDistribution<T> {
         Self::Gaussian(Gaussian::from_standard_dev(std, 0.0))
     }
 
-    pub fn new_gaussian(std: impl DispersionParameter) -> Self {
+    pub fn new_gaussian(dispersion: impl DispersionParameter) -> Self {
         Self::Gaussian(Gaussian::from_standard_dev(
-            StandardDev(std.get_standard_dev()),
+            StandardDev(dispersion.get_standard_dev()),
             0.0,
         ))
     }
 
     pub const fn new_t_uniform(bound_log2: u32) -> Self {
         Self::TUniform(TUniform::new(bound_log2))
     }
+
+    #[track_caller]
+    pub fn gaussian_variance(&self) -> Variance {
+        match self {
+            Self::Gaussian(gaussian) => {
+                Variance(StandardDev::from_standard_dev(gaussian.std).get_variance())
```

Comment:
Not introduced in this PR, but `get_value` could return a `Variance`
Same for other `DispersionParameter` methods

Reply:
not sure I understand ?

Reply:
ah yes got it, I'll have a follow up issue !

Reply:
https://github.com/zama-ai/tfhe-rs-internal/issues/485